packages/block-editor/src/components/inserter/block-types-tab.native.js

```diff
@@ -10,28 +10,16 @@ import BlockTypesList from '../block-types-list';
 import useClipboardBlock from './hooks/use-clipboard-block';
 import { store as blockEditorStore } from '../../store';
 import useBlockTypeImpressions from './hooks/use-block-type-impressions';
-
-const NON_BLOCK_CATEGORIES = [ 'reusable' ];
-
-const ALLOWED_EMBED_VARIATIONS = [ 'core/embed' ];
+import { filterInserterItems } from '.utils';
```

Comment:
Looks like we're missing a `/` here:
```suggestion
import { filterInserterItems } from './utils';
```